### PR TITLE
Swteam/#472/change wording of matches

### LINKED ISSUE #472: Change wording of matches
**AS** a User
**I WANT TO** have a different wording for the settings of a Match
**TO** better understand the different fields

**ACCEPTANCE CRITERIA**

- [x] The label Gameday should be german which would be "Spieltag"
image 2018-01-30 14 35 01

- [x] In the properties of a game the labels should not be "Score home" and "Score away" but be the players names
image 2018-01-30 14 31 53

- [x] The scores of a game should be named "Punkte". The current points should be named "Ligapunkte"
example:
image 2018-01-30 14 32 05

### PR DESCRIPTION
**Pull request by DanielStephan97**
closes #472

### CODE REVIEW

File: app/views/matches/_edit_results.html.erb [hunk 1]
```diff
@@ -11,13 +11,13 @@
       <%= f.nested_fields_for :game_results do |builder| %>
```
Comment (ErikBrendel) [inline app/views/matches/_edit_results.html.erb]: why `builder` and not `f` as usual?

File: app/views/matches/_edit_results.html.erb [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
         <div class="result">
           <div class="form-group">
-            <%= builder.label :score_home, GameResult.human_attribute_name(:score_home), class: 'control-label col-xs-2' %>
+            <%= builder.label :score_home, t('view.match.game_result.points')+ ' ' + @match.team_home.name, class: 'control-label col-xs-2' %>
```
Comment (ErikBrendel) [inline app/views/matches/_edit_results.html.erb]: spacing around the concat operator please

File: app/views/matches/_edit_results.html.erb [hunk 3]
```diff
@@ -11,13 +11,13 @@
...
             <div class="col-xs-2">
               <%= builder.number_field :score_home, in: -2e9..2e9, step: 1, class: 'form-control', required: :true %>
             </div>
           </div>
           <div class="form-group">
-            <%= builder.label :score_away, GameResult.human_attribute_name(:score_away), class: 'control-label col-xs-2',required: :true %>
+            <%= builder.label :score_away, t('view.match.game_result.points')+ ' ' + @match.team_away.name, class: 'control-label col-xs-2',required: :true %>
```
Comment (ErikBrendel) [inline app/views/matches/_edit_results.html.erb]: spacing here aswell

File: config/locales/de.yml [hunk 4]
```diff
@@ -196,9 +197,9 @@ de:
           team: "Team"
         bestof_length: "Spielart"
         game_winrule: "Gewinnregel"
-        points_for_win: "Punkte für Gewinn"
-        points_for_draw: "Punkte für Unentschieden"
-        points_for_lose: "Punkte für Niederlage"
+        points_for_win: "Ligapunkte für Gewinn"
```
Comment (ErikBrendel) [inline config/locales/de.yml]: we should definitely talk to the PO again here: This is `event.points_for_win`. Not every event is a league.

File: config/locales/de.yml [hunk 5]
```diff
@@ -327,6 +330,7 @@ de:
       away: "Team 2"
       score: "Ergebnis"
       points: "Punkte"
+      league_points: "Ligapunkte"
```
Comment (ErikBrendel) [inline config/locales/de.yml]: This is really weird, when you have a tournament match.

**Review by Markus-Brand — APPROVED** [review]
I agree with everything @ErikBrendel said.
So i approve, as soon as his feedback was addressed

File: config/locales/de.yml (hunk 4 above)
Comment (Markus-Brand) [reply config/locales/de.yml]: Rename to Eventpunkte / Eventpoints please. (PO agrees)

**Pull request merged**